docs/user/install.md

```diff
@@ -0,0 +1,15 @@
+# Installation instructions
+
+This document outlines how to enable the pre-released version of Firefox Translations which comes bundled with Firefox Nightly.
+
+### Configuring Nightly to enable the extension
+
+- Download and install Firefox Nightly (English or German edition) from [https://www.mozilla.org/en-US/firefox/all/#product-desktop-nightly](https://www.mozilla.org/en-US/firefox/all/#product-desktop-nightly).
```

Comment:
Can it only be `English or German edition` of Firefox Nightly?

Reply:
It can also be the other supported languages

Reply:
(This user-guide section is meant to be improved by the team as they see fit)